### PR TITLE
:bug::shell: Fix GTDB taxon-binning workflow

### PR DESCRIPTION
**Pull request by shaneroesemann**
Implement changes by @Sidduppal in issue https://github.com/KwanLab/Autometa/pull/329#issuecomment-1594996732

### CODE REVIEW

**Review by Sidduppal — APPROVED** [review]
Looks good.

File: workflows/autometa-large-data-mode.sh [hunk 1]
```diff
@@ -241,10 +241,10 @@ then
         set -x
```
Comment (WiscEvan) [inline workflows/autometa-large-data-mode.sh:241]: Would you mind adding a comment of an example header?

```suggestion
        # <insert example ORF header>
        set -x
```

File: workflows/autometa.sh [hunk 2]
```diff
@@ -231,10 +231,10 @@ then
         set -x
```
Comment (WiscEvan) [inline workflows/autometa.sh:231]: Would you mind adding a comment of an example header?

```suggestion
        # <insert example ORF header>
        set -x
```

File: workflows/autometa-large-data-mode.sh [hunk 3]
```diff
@@ -241,10 +241,10 @@ then
...
         grep ">" $kingdom_fasta | \
             sed 's/^>//' | \
-            sed 's/$/_/' | \
-            cut -f1 -d" " > $orf_prefixes
+            cut -f1 -d" " | \
+            sed 's/$/_/' > $orf_prefixes
         # Retrieve ORF IDs from contig IDs
-        grep -f $orf_prefixes $orfs | sed 's/^>//' | cut -f1 -d" " > $orf_ids
+        grep -f $orf_prefixes $orfs | cut -f1 -d" " | sed 's/^>//'  > $orf_ids
```
Comment (WiscEvan) [inline workflows/autometa-large-data-mode.sh:247]: Why must the order of `sed` and `cut` be swapped in this line?

:art::fire: Removing unnecessary whitespace

```suggestion
        grep -f $orf_prefixes $orfs | cut -f1 -d" " | sed 's/^>//' > $orf_ids
```

File: workflows/autometa.sh [hunk 4]
```diff
@@ -231,10 +231,10 @@ then
...
         grep ">" $kingdom_fasta | \
             sed 's/^>//' | \
-            sed 's/$/_/' | \
-            cut -f1 -d" " > $orf_prefixes
+            cut -f1 -d" " | \
+            sed 's/$/_/' > $orf_prefixes
         # Retrieve ORF IDs from contig IDs
-        grep -f $orf_prefixes $orfs | sed 's/^>//' | cut -f1 -d" " > $orf_ids
+        grep -f $orf_prefixes $orfs | cut -f1 -d" " | sed 's/^>//'  > $orf_ids
```
Comment (WiscEvan) [inline workflows/autometa.sh:237]: Why must the order of sed and cut be swapped in this line?

**Review by WiscEvan — COMMENTED** [review]
Would you mind updating the description for how these changes resolve the issue? Currently this is somewhat unclear. Not saying that the issue has not been resolved, it is just difficult to discern with the information that is currently present. Is there a specific instance where partial matching was causing a problem?

File: workflows/autometa.sh (hunk 4 above)
Comment (chasemc) [reply workflows/autometa.sh:237]: Probably doesn't matter much but might be faster since there's less data being piped to the last step
